core/templates/dev/head/pages/base.html

```diff
@@ -177,6 +177,8 @@ <h2>
     <script src="/templates/dev/head/i18n.js"></script>
 
     <script src="/templates/dev/head/pages/Base.js"></script>
+    <script src="/templates/dev/head/pages/suggestion_editor/ShowSuggestionModalForLearnerLocalViewService.js"></script>
```

Comment:
Can we keep this sorted? This will go below the other js file then.

Reply:
By the way, why are we importing this in base.html? Can we keep it in some html where it is more specific?

Reply:
Done.